pennylane/devices/qutrit_mixed/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+Submodule for performing mixed-qutrit-based simulations of quantum circuits.
+
+This submodule is internal and subject to change without a deprecation cycle. Use
+at your own discretion.
+
+.. currentmodule:: pennylane.devices.qutrit
+.. autosummary::   TODO is this necessary?
```

Comment:
```suggestion
.. autosummary::
```

This is used to create an automatic summary in the PennyLane documentation. Adding this will allow us to add the qutrit helpers to the `qml.devices` section in the future. All we would have to do is add the following to `pennylane/devices/__init__.py`:
```
.. currentmodule:: pennylane.devices.qutrit
.. automodule:: pennylane.devices.qutrit
```

Reply:
To see what this could do, see: https://docs.pennylane.ai/en/stable/code/qml_devices.html#module-pennylane.devices.qubit

But the importance of this will come down to whether we want to show all these functions in the website build.  All of this can be resolved later down the line as a final step to pull the project together.

Reply:
@albi3ro what should we have for now?

Reply:
removed and added a TODO for later